charts/velocity-operator/values.yaml

```diff
@@ -56,6 +56,9 @@ aws:
     k8sSecretKeyName: &k8s-secret-key-name credentials
     profile: &aws-profile default
 
+velocity:
+  apiUrl: ""
```

Comment:
Let's specify the default value for this to help the user

Reply:
This is not for the user but internally for us to use dev/prod. I don't want to specify our API URL here